### DISCUSSION TITLE
Change font for different language

### DISCUSSION THREAD

**Discussion by muhammadidrees** [cstayyab/WALC#72] [Feature Requests]
Currently there in no way (that I know of) to change font of different language. I receive text in Urdu and it is rendered horibally almost impossible to read.

Now that issue is resolved in the browser by using extensions to render urdu font differently but WALC doesn't seem to support such feature.

Please add this feature!

**Comment by daffamumtaz2361** [comment]
I don't uderstand, what do you mean by rendered horribly? Does this also happen with whatsapp web? But I guess I could implement font selector that you could use to override the default font with the font installed on your system.

**Comment by muhammadidrees** [comment]
I mean the font used by default doesn't support all characters of the Urdu language.

![Screenshot from 2020-11-21 18-27-03](https://user-images.githubusercontent.com/36603847/99878457-47824b00-2c27-11eb-8a62-9718697c6cba.png)

Now a workaround I did was adding the Urdu Language supported font in the CSS body styling via the debug tools.

![Screenshot from 2020-11-21 18-31-50](https://user-images.githubusercontent.com/36603847/99878560-19513b00-2c28-11eb-94e6-f4de9aac91c3.png)

And now it is rendering properly

![Screenshot from 2020-11-21 18-33-52](https://user-images.githubusercontent.com/36603847/99878570-30902880-2c28-11eb-8ec0-f6302eb83fe8.png)

The solution is not permanent though it if I restart my laptop all of the settings are gone and I have to re-enter the supported font in CSS styles again.

So like you said it would be nice to have a font selector or any easier way of making this change and also keeping it permanent so it doesn't go away on restarts - as per one's need.

Hope this further clarifies the issue I am addressing. :grin:

**Comment by muhammadidrees** [comment]
@Zzombiee2361 with the latest update I can't even access the developer tools to change the font.

Can you please fix this!!!
